src/settings_data.h

```diff
@@ -586,6 +587,17 @@ static const struct setting allowed_settings[] = {
                 .parser_data = NULL,
                 .rule_offset = offsetof(struct rule, set_icon_size),
         },
+        {
+                .name = "enable",
```

Comment:
I would name it "enabled"

Reply:
Done with https://github.com/dunst-project/dunst/pull/981/commits/f4e445b7f6c4df6e907a4f9bdf597ca24ca696e2, should I rename it elsewhere?

Reply:
Could you rename it in the rule struct and it's usages as well?

Reply:
Thats easy. Done in https://github.com/dunst-project/dunst/pull/981/commits/1440a0d94ad3950fefb92b454babf7e2de9ffe6e .

Reply:
Thanks!